src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -285,6 +287,19 @@ public void Initialize(InitializeOptions options = InitializeOptions.None)
                         // We ignore the exception since it was caught in the function itself
                     }
                 }, null);
+
+                Task.Run(() =>
+                {
+                    try
+                    {
+                        ExectueClusterTransactionTask();
+                    }
+                    catch
+                    {
+                        //
```

Comment:
??

Reply:
Do we even need this `try/catch`? since the `ExectueClusterTransactionOnDatabase` function is already wrapped with `try/catch`.

Reply:
Probably can remove that, yes.